stock_release_channel_show_volume/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2023 Jacques-Etienne Baudoux (BCIM) <je@bcim.be>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+{
+    "name": "Stock Release Channels show Volume",
+    "summary": "Display volumes of stock release channels",
+    "version": "16.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "MT-Software, BCIM, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/wms",
+    "depends": [
+        "stock_picking_volume",
```

Comment:
About `stock_picking_volume`, I forward-ported the work done by @mt-software-de on 14.0 here:
- https://github.com/OCA/stock-logistics-warehouse/pull/1954